packages/main/test/specs/RatingIndicator.spec.js

```diff
@@ -122,4 +122,18 @@ describe("Rating Indicator general interaction", () => {
 
 		assert.strictEqual(await ratingIndicator.getAttribute("title"), TOOLTIP, "The title attribute is rendered in the inner div as well.");
 	});
+
+	it("Tests ACC attrs - required property add aria-description", async () => {
+		const ratingIndicator = await browser.$("#rating-indicator-required");
+		const REQUIRED = "required";
```

Comment:
as the text should be translated you should test against the resourse bundle text (see other tests with translations)

Reply:
Done